spec/lib/protobuf/field/message_field_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+RSpec.describe Protobuf::Field::MessageField do
+  class MessageField < ::Protobuf::Message
```

Comment:
just as a heads up, this will leak the name `MessageField` into the global namespace in all specs, see `Class.new` in things like `enum_spec.rb` and `message_spec.rb` for how to define anonymous classes in tests that don't leak?

it's not the end of the world but a good practice IMO in case some other test tries to make a class named `MessageField`